packages/jbrowse-plugin-apollo/src/ApolloTextSearchAdapter/ApolloTextSearchAdapter.ts

```diff
@@ -0,0 +1,72 @@
+import { readConfObject } from '@jbrowse/core/configuration'
+import {
+  BaseAdapter,
+  BaseTextSearchAdapter,
+  BaseTextSearchArgs,
+} from '@jbrowse/core/data_adapters/BaseAdapter'
+import BaseResult from '@jbrowse/core/TextSearch/BaseResults'
+import { UriLocation } from '@jbrowse/core/util'
+import { getFetcher } from '@jbrowse/core/util/io'
+
+export class ApolloTextSearchAdapter
+  extends BaseAdapter
+  implements BaseTextSearchAdapter
+{
+  get baseURL() {
+    return readConfObject(this.config, 'baseURL').uri
+  }
+
+  get trackId() {
+    return readConfObject(this.config, 'trackId').uri
```

Comment:
No `uri` property here
```suggestion
    return readConfObject(this.config, 'trackId')
```